app/views/partials/a-propos.html

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <h1>A propos</h1>
+  <h1 tabindex="-1">A propos</h1>
```

Comment:
I chose not to do it to handle it differently in some cases (eg. focus on birth date). I'll add it dynamically and add some sort of exception.